### ISSUE TITLE
Many mobs are not moving

### ISSUE THREAD

**Issue by ioicomeone** [PaperMC/Paper#4216]
### What behaviour is expected:
After updating to 1.16.2 Mobs moving about as normal. 

### What behaviour is observed:
After updating to 1.16.2, Mobs have audio and can be interacted with, but do not otherwise move. New mobs that spawn, i.e zombies, creepers, skeletons, spiders, chickens hatched from eggs, etc all move as normal. Also mobs that were in a further base move as normal. But a majority of sheeps, cows, horses, villagers and iron golems in our base do not move. The problem persisted after restoring a backup and downgrading back to Paper 1.16.1 on Apex servers.

### Steps/models to reproduce: 
We were playing on Apex servers using Paper 1.16.1. 
Update to 1.16.2
Start game
Observed issues.
Downgrade to 1.16.1
Restore Backup from last night before the update.
Start game
Observe issues

### Plugin list:
WorldBorder 1.8.7

### Paper version:
This server is running Paper version git-Paper-138 (MC: 1.16.1) (Implementing API version 1.16.1-R0.1-SNAPSHOT)
Checking version, please wait...
Previous version: git-Paper-145 (MC: 1.16.2)
You are running the latest version

### Anything else:
I was trying to preload chunks with WorldBorder previously. 
The bug seems to happen mainly in the general vicinity of our intial world spawn point.

**Comment by Proximyst** [comment]
We do not support any issues caused by and through downgrades.

Did you use EntityTrackerFixer before this?
Share your configuration files.

**Comment by ioicomeone** [comment]
The issue occurred after upgrading. It persisted through the downgrade. 

We have tried using EntityTrackerFixer! I believe we removed it but we had tried it briefly on the server to help fix some lag. 

Apologies, I'm not sure if these are what you are looking for but these are what I could find clicking around the panel.

EntityTrackerFixer config.yml
-----------------------------------------------
              EntityTrackerFixer

                By: Esmorall

    English of this file improved by ArtelGG
-----------------------------------------------

 Should we notify the console when we interact with entities?
 - Default value(s): true
log-to-console: true

 Should we disable tick operations of un-tracked entities?
 Note 1: this option only works for +1.16.1 servers
 Note 2: this option is experimental and could contain errors
 - Default value(s): true
disable-tick-for-untracked-entities: true

 Should the plugin perform tasks on all worlds?
 Note: if this is set to true, the option "worlds" will be ignored
 - Default value(s): false
enable-on-all-worlds: false

 How often (in ticks) should we check for "lingering" entities?
 - Default value(s): 400
untrack-ticks: 400

 How low should the server's TPS be before we do anything?
 - Note: Setting this value above 20 will skip this check, allowing the tasks to run 24/7.
 - Default value(s): 19.5
tps-limit: 19.5

 How often (in ticks) should we check for players near un-tracked entities?
 - Default value(s): 40
check-untracked-entities-frequency: 40

 How far (in blocks) should we look for players near un-tracked entities?
 - Default value(s): 30
tracking-range: 30

 What worlds should we perform our tasks on?
 - Default value(s): [world, world_nether, world_the_end]
worlds:
  - world
  - world_nether
  - world_the_end

Paper.yml
 This is the main configuration file for Paper.
 As you can see, there's tons to configure. Some options may impact gameplay, so use
 with caution, and make sure you know what each option does before configuring.
 
 If you need help with the configuration or have any questions related to Paper,
 join us in our Discord or IRC channel.
 
 Discord: https://paperdiscord.emc.gs
 IRC: #paper @ irc.spi.gt ( http://irc.spi.gt/iris/?channels=paper )
 Website: https://papermc.io/ 
 Docs: https://paper.readthedocs.org/ 

verbose: false
config-version: 20
settings:
  max-joins-per-tick: 3
  player-auto-save-rate: -1
  max-player-auto-save-per-tick: -1
  load-permissions-yml-before-plugins: true
  bungee-online-mode: true
  region-file-cache-size: 256
  incoming-packet-spam-threshold: 300
  console-has-all-permissions: false
  suggest-player-names-when-null-tab-completions: true
  use-alternative-luck-formula: false
  enable-player-collisions: true
  chunk-tasks-per-tick: 1000
  save-empty-scoreboard-teams: false
  save-player-data: true
  velocity-support:
    enabled: false
    online-mode: false
    secret: ''
  watchdog:
    early-warning-every: 5000
    early-warning-delay: 10000
  spam-limiter:
    tab-spam-increment: 1
    tab-spam-limit: 500
  book-size:
    page-max: 2560
    total-multiplier: 0.98
  unsupported-settings:
    allow-headless-pistons: false
    allow-permanent-block-break-exploits: false
    allow-piston-duplication: false
    allow-permanent-block-break-exploits-readme: This setting controls if players
      should be able to break bedrock, end portals and other intended to be permanent
      blocks.
    allow-piston-duplication-readme: This setting controls if player should be able
      to use TNT duplication, but this also allows duplicating carpet, rails and potentially
      other items
    allow-headless-pistons-readme: This setting controls if players should be able
      to create headless pistons.
  async-chunks:
    enable: true
    threads: -1
messages:
  no-permission: '&cI''m sorry, but you do not have permission to perform this command.
    Please contact the server administrators if you believe that this is in error.'
  kick:
    authentication-servers-down: ''
    connection-throttle: Connection throttled! Please wait before reconnecting.
    flying-player: Flying is not enabled on this server
    flying-vehicle: Flying is not enabled on this server
timings:
  enabled: true
  verbose: true
  server-name-privacy: false
  hidden-config-entries:
  - database
  - settings.bungeecord-addresses
  history-interval: 300
  history-length: 3600
  server-name: Unknown Server
world-settings:
  default:
    baby-zombie-movement-modifier: 0.5
    optimize-explosions: false
    disable-teleportation-suffocation-check: false
    fixed-chunk-inhabited-time: -1
    use-vanilla-world-scoreboard-name-coloring: false
    remove-corrupt-tile-entities: false
    experience-merge-max-value: -1
    prevent-moving-into-unloaded-chunks: true
    max-auto-save-chunks-per-tick: 6
    count-all-mobs-for-spawning: false
    per-player-mob-spawns: true
    falling-block-height-nerf: 0
    tnt-entity-height-nerf: 0
    filter-nbt-data-from-spawn-eggs-and-related: true
    max-entity-collisions: 2
    disable-creeper-lingering-effect: false
    duplicate-uuid-resolver: saferegen
    duplicate-uuid-saferegen-delete-range: 32
    phantoms-do-not-spawn-on-creative-players: true
    phantoms-only-attack-insomniacs: true
    keep-spawn-loaded: false
    grass-spread-tick-rate: 4
    water-over-lava-flow-speed: 5
    bed-search-radius: 1
    nether-ceiling-void-damage-height: 0
    allow-non-player-entities-on-scoreboards: false
    portal-search-radius: 128
    portal-create-radius: 16
    container-update-tick-rate: 3
    parrots-are-unaffected-by-player-movement: false
    disable-explosion-knockback: false
    disable-thunder: false
    skeleton-horse-thunder-spawn-chance: 0.01
    disable-ice-and-snow: false
    keep-spawn-loaded-range: 10
    armor-stands-do-collision-entity-lookups: true
    prevent-tnt-from-moving-in-water: false
    iron-golems-can-spawn-in-air: false
    armor-stands-tick: false
    non-player-arrow-despawn-rate: 60
    creative-arrow-despawn-rate: 60
    spawner-nerfed-mobs-should-jump: false
    entities-target-with-follow-range: false
    zombie-villager-infection-chance: -1.0
    all-chunks-are-slime-chunks: false
    mob-spawner-tick-rate: 2
    delay-chunk-unloads-by: 10s
    light-queue-size: 20
    auto-save-interval: -1
    enable-treasure-maps: true
    treasure-maps-return-already-discovered: false
    game-mechanics:
      scan-for-legacy-ender-dragon: true
      disable-pillager-patrols: false
      disable-chest-cat-detection: true
      nerf-pigmen-from-nether-portals: false
      disable-player-crits: false
      disable-sprint-interruption-on-attack: false
      disable-end-credits: false
      shield-blocking-delay: 5
      disable-unloaded-chunk-enderpearl-exploit: true
      disable-relative-projectile-velocity: false
      pillager-patrols:
        spawn-chance: 0.2
        spawn-delay:
          per-player: false
          ticks: 12000
        start:
          per-player: false
          day: 5
    max-growth-height:
      cactus: 3
      reeds: 3
    fishing-time-range:
      MinimumTicks: 100
      MaximumTicks: 600
    despawn-ranges:
      soft: 28
      hard: 96
    frosted-ice:
      enabled: true
      delay:
        min: 20
        max: 40
    lootables:
      auto-replenish: false
      restrict-player-reloot: true
      reset-seed-on-fill: true
      max-refills: -1
      refresh-min: 12h
      refresh-max: 2d
    alt-item-despawn-rate:
      enabled: true
      items:
        COBBLESTONE: 300
    hopper:
      cooldown-when-full: true
      disable-move-event: true
    lightning-strike-distance-limit:
      sound: -1
      impact-sound: -1
      flash: -1
    anti-xray:
      enabled: true
      engine-mode: 1
      max-chunk-section-index: 3
      update-radius: 2
      hidden-blocks:
      - gold_ore
      - iron_ore
      - coal_ore
      - lapis_ore
      - mossy_cobblestone
      - obsidian
      - chest
      - diamond_ore
      - redstone_ore
      - clay
      - emerald_ore
      - ender_chest
      replacement-blocks:
      - stone
      - oak_planks
    viewdistances:
      no-tick-view-distance: -1
    squid-spawn-height:
      maximum: 0.0
    generator-settings:
      flat-bedrock: false

**Comment by Proximyst** [comment]
The issue is most likely caused by ETF, which is why new mobs work just fine.

**Comment by ioicomeone** [comment]
Ah I see. I don't suppose you would be able to suggest what might be causing the issue and if removing the ETF config files would fix it? I remember the reason we removed it this morning was because it was causing an error during startup on 1.16.2, which looks now like the likely cause for our problems.

**Comment by Octol1ttle** [comment]
config files cant do anything without something using them

**Comment by Octol1ttle** [comment]
also btw dont use etf on paper, otherwise it will do more harm than good
Try `/execute as @e[type=!player] run data merge entity @s {NoAI:0}` (Not sure it will work)

**Comment by ioicomeone** [comment]
> config files cant do anything without something using them

Ah of course. Very true. Sorry, I don't have much experience with config files or plugins.



> also btw dont use etf on paper, otherwise it will do more harm than good
> Try `/execute as @e[type=!player] run data merge entity @s {NoAI:0}` (Not sure it will work)

Thanks! This is good to know! I was following suggestions by the tech support but it seems that they didn't realize this. 
I just tried to run that in game. It gave me a list of things it modified, but it didn't help with Villagers, and unfortunately also with not the chickens. Still frozen even though the text says 'Modified entity data of Chicken'

**Comment by Octol1ttle** [comment]
maybe replacing `!player` with `villager` and `chicken` and spamming it a few times might help
these are only my thoughts & suggestions tho

**Comment by ioicomeone** [comment]
Thank you very much for the suggestion. They're still stuck after I tried that but thanks! I was looking for a way to maybe refresh the AI because they seem like they're just stuck in a frozen state. I can still trade and interact with them for the most part but they just forgot how to move. I noticed that ETF has had this frozen mob issue previously so maybe I can send them a message and see if they have a fix for it.
